robottelo/cli/base.py

```diff
@@ -374,6 +374,13 @@ def _construct_command(cls, options=None):
                 if val is True:
                     tail += u' --{0}'.format(key)
                 elif val is not False:
+                    if isinstance(val, list):
+                        val = ','.join(
+                            str(el.get('id') or el.get('name'))
```

Comment:
:+1:

It's common to pass in names or IDs, but even a single violation of this pattern rule breaks this logic. Who knows - maybe labels will be required somethere, or maybe UUIDs?